image-builder/README.md

```diff
@@ -14,12 +14,97 @@ We use the Docker's building system and convert it at the end to a disk image th
 [More info](https://github.com/scaleway/image-tools)
 
 
-:warning: **Documentation incoming** :warning:
+## How to build a custom image (using [scw](https://github.com/scaleway/scaleway-cli))
+
+**My custom image's description**
+- based on [ubuntu vivid](https://github.com/scaleway/image-ubuntu)
+- `cowsay` pre-installed
+
+---
+
+##### 1 Making the environment
+
+```bash
+root@yourmachine> scw run --rm --name="buildcowsay" builder
+...SKIP
```

Comment:
Do you really need to skip the content ? I think a `scw run` without `-D/--debug` option will be small enough to avoid skipping content